Final Report: Life After Death- Identifying microbiotic signatures in decomposing cadavers
================
Alfredo H. Locht
14Dec2017

Introduction
============

The decomposition of the human body can yield many clues which forensic scientists and law enforcement use (Hauther *et al.*, 2015) to solve criminal investigations. In the first four minutes after a human dies, the body is depleted of oxygen and carbon dioxide begins to dissolve in the blood, creating carbonic acid and lowering the pH of the blood, causing cells to lyse (autolysis) and release their contents (Vass, 2001, 2011). These include many enzymes such a proteases and lipases which have digestive activities that begin the initial breakdown of bodily tissue and organs (Vass, 2001; Distinctive thanatomicrobiome signatures found in the blood and internal organs of humans, 2014; Cockle and Bell, 2015). Body organs which have high enzymatic activity such as the liver, or provide an aqueous environment that aids in providing the necessary ions for enzymatic catalysis such as the brain, tend to decompose faster than other organs, so there is a wide variability of rates at which specific sites on the body tends to decompose; however, the body can be characterized at each of the different stages of decomposition in summation (Hyde *et al.*, 2013, 2015; Distinctive thanatomicrobiome signatures found in the blood and internal organs of humans, 2014). The five stages of human decomposition include: Fresh which will be termed Autolysis, Putrefaction/Bloat, Purge (Active Decay), Advanced Decay and Skeletonization. The rate at which is one occurs or lasts is highly dependent on a myriad of factors that up to date, have yet not been efficiently addressed (Hyde *et al.*, 2013, 2015).

Characterization of each step in the decomposition process is critical to law enforcement, forensic scientists and many other disciplines alike because they use these markers to reveal information about the Post Mortem Interval (PMI), or better known as the time since death (Comparison of decomposition rates between autopsied and non-autopsied human remains, 2016; Ferreira and Cunha, 2013; Hauther *et al.*, 2015; Hyde *et al.*, 2013). The time since death is a key contributor to solving investigations surrounding death because it will provide a c The PMI is obtained in a variety of ways using a variety of methods, but has previously relied on visual markers, chemical profile analyses, measurement of enzymatic activity, carbon dioxide-carbon evolution and other predominantly visual or chemically-based methods of analyses (Carter *et al.*, 2008; Ferreira and Cunha, 2013; Hauther *et al.*, 2015; Rosier *et al.*, 2015; Vass, 2011; <span class="citeproc-not-found" data-reference-id="den">**???**</span> Besten2013; Ríos-Covián *et al.*, 2016).

Each method of course, has its corresponding error rate which can overestimate or underestimate the PMI and lead to erroneous results (Cockle and Bell, 2015). For example, very frequently, validation of a specific PMI calculation derived by one set of researchers in a particular geographic location is only valid at that particular location because when tested in significantly different ecological and temperature climate, the studies cannot match the results expected or previously reported (Cockle and Bell, 2015). Furthermore, researchers often argue about which factors directly influence the PMI, so a consensus on how to develop methods which would measure certain sets of factors, has not been reached (Carter *et al.*, 2008; Cockle and Bell, 2015; Ferreira and Cunha, 2013). It is also cumbersome that the identified list of potential factors influencing PMI estimations include endogenous factors such as body and fat mass, penetrating trauma, drug use, disease profile, and clothing as well as exogenous factors such as temperature, vegetation in the area, humidity, type of soil, and scavenger and insect access all compile to precisely paint the rate and pattern of a decomposition picture (Comparison of decomposition rates between autopsied and non-autopsied human remains, 2016; Carter *et al.*, 2008; Cockle and Bell, 2015; Hauther *et al.*, 2015). What this picture is missing however, are the organisms directly responsible for the process.

Interestingly, microbial communities have been outright dismissed by certain forensic scientists as having anything to do with the decomposition process of human corpses (Carter *et al.*, 2008), yet other sources explain that microbial decomposers are considered to have co-evolved with the presence of vertebrate corpses for the need to recycle complex biological molecules back into basic compounds and elements (Metcalf *et al.*, 2016). It is currently acknowledged, however, by many sources that there exists a distinct change in the microbial communities decomposing the human body throughout the putrefaction stage, characterized by a shift from dominance exhibited by aerobic bacteria to dominance exhibited by anaerobic bacteria (Hyde *et al.*, 2013, 2015; Emmons *et al.*, 2017). This is possibly due to the fact that as the present microbes are mostly aerobic at the beginning of putrefaction, they begin to produce various gaseous products which "bloat" the corpse. Furthermore, the gaseous exchanges between the interior of the body and the exterior no longer happen, so what results is that the aerobic bacteria bascially poison themselves with all of the waste they develop during decomposition. This creates room for facultive and obligate anaerobes to proliferate in an environment that is oxygen-deficient. It is not until the holes carved by other scavengers "purge" the body of the gaseous and liquid buildup, that the cadaver can become an ecosystem to aerobic bacteria once more.

In this study, the shift will be measured by the relative abundance of the following bacteria to characterize the transition. Specifically, it is expected that microbes from the *Bacilli* genera will decrease in abundance while microbes from the *Bacteroidetes* phyla will increase in abundance as the samples go from pre-bloat to end-bloat (Hyde *et al.*, 2013). Furthermore, since (Javan *et al.*, 2017) discusses the predominance of the *Clostridia* class in the decomposition of the body, these microbes will be also expected to increase in all parts of the cadaver.

Methods
=======

Study design
------------

(Hyde *et al.*, 2013) obtained cadavers donated to the Southeast Texas Applied Forensic Science (STAFS) Facility at the Center for Biological Field Studies (CBFS) at Sam Houston State University (SHSU). Both CBFS and STAFS are located in a humid, subtropical climate characterized by a large distribution of pine trees and acidic soils in an area 5 km North of Huntsville, Texas. STAFS is a sectioned-off, two-acre area within the larger, 243 acre CBFS complex which is covered with pine trees and herbaceous plants. The STAFS facility is considered to be a “willed donations” center, so donors or family members of the deceased are required to sign a form in which they indicate that they understand that their donated corpse will be used for scientific research and teaching purposes.

This form also included information regarding the methods that would be used to study the cadavers. Specifically, the study informed the donors and/or next of kin that the body would be left outside to decompose in natural and aerobic conditions, and that samples pertaining to different types of research (bacterial, tissue, and entomological) would be acquired at different stages of the decomposition to support the associated research. Upon request, the family members could also choose to be debriefed about the scientific knowledge or educational project that was achieved during the course of the study and as a result of the donated cadaver.

Sample Origin and Sequencing
----------------------------

### Cadaver Demographics

The two cadavers donated to the center that were studied, STAFS 2011-016 and STAFS 2011-006, were both males of ages 68 and 52 respectively, both of Caucasian descent, were corpses that had not been autopsied, and were considered to be in the early stages of decomposition according to a qualitative body-scoring index developed by (Megyesi *et al.*, 2005). This index relied on visual observation of the cadavers for signs of the bloat process (Hyde *et al.*, 2013). STAFS 2011-016 had a previous medical history relating to the cause of death (acute myocardial infarction), been frozen for 89 days, and had been subsequently defrosted in a fridge cooler for 143 days before being placed outside to decompose. STAFS 2011-006 did not have any pertaining medical history other than the cause of death (carbon monoxide poisoning), had been frozen for 13 days, and had subsequently defrosted for 6 days in a fridge cooler before decomposition placement. The time since death was unknown for both cadavers.

### Sampling Methods

Bacterial sampling of each cadaver was obtained from multiple body sites at two different time points, Placement (Pre-Bloat Phase) and Conclusion (End-Bloat Phase). The Pre-Bloat samples were taken right before the corpse was placed in a supine position outside with no clothing to decompose in natural conditions, while the End-Bloat samples required that the cadaver be dissected in order to gain access to the internal organs. Because this procedure was intrusive and compromised the structural integrity of the cadaver, it marked the end time-point of the sampling.

Pre-bloat samples were taken as either swabs or “scrapes” from the mouth and rectum by swabbing/scraping with a sterile cotton applicator which was subsequently cut into a sterile PBS solution and prepped for sequencing using the PowerSoil DNA Isolation Kit (MoBio). Bloat samples were taken from the corpses at a time which they were identified to have purged their bodily fluids according to the body scoring system (Megyesi *et al.*, 2005). Maggots found on the cadavers were also removed at this point with sterile plastic spatulas for sampling purposes.

The samples obtained from different parts of the body or organs were sampled whenever those body parts or organs still existed. Specifically, STAFs 2011-016 no longer retained any evidence of a sigmoidal colon at the end-bloat sampling.

Computational
=============

The data used for this project was produced by downloading 16S amplicon data from SRA study number: SRP031900 and the respective table of metadata. Next, the SRA toolkit was downloaded from NCBI in order to perform a fastq dump to work with the files on a personal computer.

After downloading the files with the SRA toolkit, the files were processed through the DADA2 pipeline. This pipeline creates a working script to create a phyloseq object, which is then used to analyze comprehensive sample sets that are directly tied to their metadat, OTU classification, and oftentimes, Phylogenetic tree. Both forward and reverse reads are plotted with their respective quality profiles, trimming on either side with the FilterandTrim function. For the purposes of this study, only the forward reads were used. The FilterandTrim parameters are listed below: maxLen = 225, maxN = 0, maxEE = 3, truncQ = 2, rm.phix = TRUE, compress = TRUE, multithread = FALSE

Following FilterandTrim, DADA2 relies on the parametric error model to calculate the expected error in each of the reads acquired. The DADA2 plotErrors function will predict the error model from the data, and subsequently alternates both error rate prediction and inference of sample composition until these two factors converge at a single, unfluctating solution. These estimated error rates will then be plotted along with their error models. Verification of successful error modeling will provide the user with confidence in the parameters they set for trimming data.

Next, The DADA2 pipeline will dereplicate the sequences and create a table of all unique sequence variants present in the sample set. The unique sequence variants consists of all of the identical reads and their corresponding value of reads (How many reads of a particular sequence variant exist). Once these unique sequences are identified, the chimeras present in them will be removed, so that OTU classification can be accomplished. At this point, the DADA pipeline ends, and the phyloseq package takes over to complete the creation of a phyloseq object.

Taking the output data from the DADA2 pipelne, phyloseq will combine the OTU classification table, the metadata table, and a taxonomy table all into one nice and neat data frame which can be used with the remaining tools of the Phyloseq package and visual mapping guides such as ggplot2 to create analyses and figures based on the sequencing data downloaded.

### Limting Disclaimers

Conditions surrounding the death of any given cadaver can have high variability in confounding factors which can directly influence the rate and pattern of decomposition (Comparison of decomposition rates between autopsied and non-autopsied human remains, 2016; Cockle and Bell, 2015; Ferreira and Cunha, 2013; Hauther *et al.*, 2015; Hyde *et al.*, 2013; Vass, 2011). Intrinsic factors such as cause of death, physical trauma, and postmortem care can directly interact with a number of other, extrinsic factors such as temperature and humidity of environment in manners which have not yet been efficiently characterized (Cockle and Bell, 2015). (Hyde *et al.*, 2013) forewarn that because of the inherent inconsistency of the STAFS willed donations program (cadavers can be obtained on a random or unexpected basis) and the associated complications in conducting decompositional studies on cadavers, data acquired for both cadavers cannot be compared directly but rather, should act as a complimentary set of data. Although the authors sought to control or reduce the amount of confounding variables to the best of their knowledge, they were able to identify some of the limiting qualities of their study. These disclaimers are investigated below:

In order to establish a baseline of decomposition of both cadavers, they were initially qualified by a visual scoring system developed by (Megyesi *et al.*, 2005). A summary of the system includes visible cues that might indicate at what step of the decomposition process a particular cadaver may be at. Using this qualitative method, both cadavers were considered to be in the early stages of decomposition as no bloat had yet occurred. This system was also used to identify the end-point of sampling when the purging of bodily fluids occurs; however, due to its qualitative nature, the cadavers did not spend an equal amount of time decomposing outside.

STAFS 2011-006 spent seven days out in September while STAFS 2011-016 spent fourteen days out in November. Because the body scoring system developed by (Megyesi *et al.*, 2005) is completely dependent on temperature, (Hyde *et al.*, 2013) retroactively calculated the heat-energy units for each cadaver which are a function of Accumulated Degree-Days (ADD), an arbitrary calculation of how much thermal energy a sample absorbs during a given time period, a maximum temperature and a chosen starting point. The results of the calculations were 197.14 heat-energy units for STAFS 2011-006 and 199.97 heat-energy units for STAFS 2011-016 when a thermal minimum of 4 degrees Celsius was chosen. Because November temperatures in Texas are less than September temperatures, it is logical to see that STAFS 2011-016 would probably have to spend twice as long outside in November to achieve the same level of decomposition as STAFS 2011-006 in September, but again, this is based on a visual and qualitative scoring index. Furthermore, the authors mention that choosing a different thermal minimum would result in differing calculations from the ones above (Hyde *et al.*, 2013).

Other variables speaking towards the physical characteristics such as height and weight of each male cadaver were unknown, but estimated to be of average height and above-average weight in comparison to an average American male (Hyde *et al.*, 2013).

Results
=======

``` r
melted_obj <- psmelt(phyloseq_obj)
```

### Figure 1: Alpha Diversity (Shannon and Simpson Index)

![](Final_Report_files/figure-markdown_github-ascii_identifiers/alpha%20diversity%20measure%20all-1.png)

Figure 1- Alpha diversity of the entire sample set, calculated by the Shannon function and Simpson index is displayed in Figure 1. Overall values for both measures of diversity are high for most samples except for in body006 and body016 cavity samples. Note the trends in the data which show that diversity for oral samples in body006 increase while diversity for oral samples in body016 decrease. Because the focus of the paper is to assess the microbial change in dominance from aerobic to bacteria to anaerobic bacteria, the oral samples for both cadavers will be important because they represent the only temporal data that is available for the sample set. No other samples besides the oral samples contain data for both pre and end-bloat stages.

### Table 1: OTUs with Abundance &gt; 1

Table 1 Description: Upon generating a psmelt data frame of the phyloseq\_obj, it was realized that many of the OTUs had an actual abundance of zero according to the parameters we established when creating the phyloseq\_obj. The psmelt object was therefore filtered to only include the 289 OTUs which had an abundance of greater than or equal to 1. This was the data used to build the remaining figures because all OTUs contained an abundance of at least 1.

### Figure 2: *Firmicutes* Phyla Abundance

![](Final_Report_files/figure-markdown_github-ascii_identifiers/abundance%20of%20firmicutes%20phyla%20across%20time-1.png)

Figure 2 Description: The abundance of the entire *Firmicutes* phyla is represented in Figure 2. This phyla was the most abundant over all the other phyla present. Of specific interest, is the fact that the phyla abundance decreases from pre-bloat to end-bloat in body006 oral samples (1--&gt;3) while slightly increasing in body016 oral samples(2--&gt;3). In general, the phyla decreases in abunance over time in both cadavers.

### Figure 3: *Firmicutes* Genera at Pre and End Bloat

![](Final_Report_files/figure-markdown_github-ascii_identifiers/changes%20in%20firmicutes%20genera-1.png)

Figure 3 Description: After analyzing the *Firmicutes* phyla abundance across sample types between both bodies, a more in-depth look at the specific genera present in the phyla at both pre-bloat and end-bloat samples was accomplished through this figure. The different cadavers can still be identified because body006 was only sampled in September while body016 was only sampled in November. Of specific interest is that all of the *Clostridium* genera (a genera of obligate anaerobes) increase in the end-bloat samples for both bodies. Conversely, both *Staphylococcus* and *Streptococcus* genera (faculative anaerobes) decrease in the end-bloat samples. Additionally, interesting the diversity of this phyla decreases overall in body016 while increasing in body006.

### Figure 4: Comparison of *Firmicutes* Classes per Sample Type

![](Final_Report_files/figure-markdown_github-ascii_identifiers/firmicutes%20classes%20of%20different%20sample%20types-1.png)

Figure 4 Description: After establishing the differences at the genus level in the *Firmicutes* phyla, it was important to see if the same trends could be found in each body on the class taxonomic level because *Streptococcus* and *Staphylococcus* are the predominant bacteria in the Bacilli class while the *Clostridium* genus predominate the *Clostridia* class. Of important in this figure, it is seen that in body006 oral samples, the abundance in *Bacilli* greatly decreases between 08Sep2011 and 15Sep2011; however, the opposite is true for body016 during the period of 03Nov2011 and 17Nov2011. In contrast, the abundance in *Chlostridia* increase for both bodies' oral samples substantially.

### Figure 5: Comparison of *Firmicutes* Classes at Pre and End Bloat

![](Final_Report_files/figure-markdown_github-ascii_identifiers/visual%20ratios%20of%20different%20classes%20in%20the%20firmicutes%20phyla-1.png)

Figure 5 Description: To better visualize the data presented in Figure 4, Figure 5 was created to visualize the change in dominance between *Clostridia* and *Bacilli* as a visual ratio where it can be seen that the *Chlostridia* (represented by green color) become a greater percentage of the entire bar as compared to the *Bacilli* (represented by salmon color) in the body006 oral samples, even when taking into account that their are more sequence abundance reads in the end-bloat sample. Although the *Bacilli* still dominate in body016 oral samples, an increase in *Clostridia* abundance is also denoted. Interestingly, the *Erysipelotrichia* abundance increases in both end-bloat samples on both corpses.

### Figure 6: *Bacilli* Class Alpha Diversity

![](Final_Report_files/figure-markdown_github-ascii_identifiers/bacilli%20class%20alph%20diversity%20over%20time-1.png)

Figure 6 Description: The alpha diversity of *Bacilli* was calculated in Figure 6 to see if it was any different from the overall diversity plotted in Figure 1. Interestingly, the oral samples maintain the same trends (increase in body006 and decrease in body016) for both Shannon and Simpson measures; however, the Shannon function alpha diversity for body016 generally decreases all across the board while the Simpson index increases overall.

### Figure 7: *Bacteroidetes* Genera Abundance

![](Final_Report_files/figure-markdown_github-ascii_identifiers/abundance%20of%20bacteroidetes%20phyla%20across%20time-1.png)

Figure 7 Description: The *Bacteroidetes* houses another genus, *Bacteroides*, that is also full of obligate anaerobes. In Figure 7, we see that the predominant genus within *Bacteroidetes* is the *Bacteroides* genus, and that it is heavily concentrated in the 15Sep2011 body006 samples which happen to be the end-bloat samples. Additionally, the genus also shows up in the 17Nov2011 end-bloat samples for body016. Unfortunately, there zero abundance for 08Sep2011 samples due to the trimming process, so they were no samples available to visualize.

### Figure 8: *Bacteroidetes* Phyla Alpha Diversity

![](Final_Report_files/figure-markdown_github-ascii_identifiers/bacteroidetes%20phyla%20shannon%20and%20simpson%20diversity-1.png)

Figure 8 Description: Because all samples from 08Sep20 were essentially removed through the trimming process, the need to assess diversity on an independent basis was alluded to in Figure 7. Shown in Figure 8, it is seen that Shannon function alpha diversity for Bacteroidetes is extremely low overall while Simpson index is high. Initial trends examined for oral samples and present in all diversity plots are, however, still maintained except in the Simpson index for body006. Diversity trends body016 decline as in the other plots.

Discussion
==========

Importance of Shannon Function and Simpson Index for Verification of Analyses Conducted
---------------------------------------------------------------------------------------

Due to the fact that the quality control process of trimming sequences unfit for analysis, it was surmised that some of the data might be skewed because of lack of normalization after trimming. Because of this reason, whenever possible, alpha diversity for the Shannon function and Simpson index was plotted to investigate whether the trimming process had affected the trends seen in pre-bloat and end-bloat samples. Importantly, it is known that the value of the Shannon function will increase as overall diversity increases when it is assumed that all samples are equally represented and randomly sampled (Protectingusnow.org, 2017). This will provide a "best case scenario" which will contrast to the Simpson index is because it is biased towards high diversity values as long as the abundance of the dominant microbes remains high.

Therefore, rather than attempting to analyze absolute values within both measures of alpha diversity, this study looked at the trends exhibited in the pre and end-bloat oral samples for both cadavers. In Figure 1, we see that overall microbial diveristy increases for body006 through both measures while diversity decreases for body016 through both measures as well. Given that both bodies were layed outside to decompose in a natural environment, it would seem illogical to expect that the microbial diversity of body 016 actually decreases over time, since there is an expected influx of new microbes looking to take advantage of the resources provied by a decomposing human body; however, sources cite that previous medical conditions (which body016 had) can greatly affect the microbiome and its subsequent colonization \[Hyde *et al.* (2013);\]. It is therefore justified to infer that the diversity of body016 was irregular to begin with, so less emphasis will be placed on its changes; however, it was expected that a positive diversity trend and a negative diversity trend would exist for body006 and body016 respectively throughout analysis of the data because there were no known factors that could sway these trends distinctively.

Bacilli and Clostridia
----------------------

After an appropriate abundance of *Firmicutes* was established by Figure 2, the genera was plotted in Figure 3. It is evident from Figure 3 that the two Bacilli kept as standards of aerobid respiration, *Staphylococcus* and *Streptococcus* decreased in both bodies with respect to the anaerobic standard, *Clostrida*, which alternatively increases from pre-bloat to end-bloat samples. This was further exemplified by Figures 4 and 5 (which are esentially the same) which show that the ratio of *Clostridia* to *Bacilli* increases in both cadavers from the pre-bloat to the end-bloat oral samples. This trend is also true for some of the other samples from body006 but less true for body016. Again, this is possibly due to the individual starting differences. There were no changes in diversity trends in each specific cadaver, so the results were deemed to be reliable.

Bacteroidetes
-------------

The *Bacteroidetes* phyla in contrast, was much more difficult to understand. For starters, it seems that the 08Sep2011 samples were completely disregarded as they do not appear in Figure 7. Although this was due to the trimming process, it prevented a temporal analysis of body006, which was unfortunate because it seemed to be a better model than body016, possibly due to either the previous disease state of body016, or even the season during which body016 decomposed at (November in Texas is much colder than September).

However, There was a high abundance for *Bacteroides* specifically in an end-bloat sample in body006, so at least there is some evidence explaining that these microbes do tend to show up towards the end of the putrefaction stage as they are obligate anaerobes.

The diversity graph in Figure 8 however, seems to be skewed towards a loss of samples from the pre-bloat stage because Shannon diversity is extremely low for the majority of samples. As a decomposing body ripe with nutrients and little to no regulation over the availability and distribution of those nutrients, the microbial communities present in this ecosystem are not subdued by limiting resources, createing an unlevel playing field in which some bacteria will effectively prove to be advantaegous and dominate over the rest (Javan *et al.*, 2017; The thanatomicrobiome: A Missing Piece of the Microbial Puzzle of Death, 2016, 2016). Such domination would be represented by high diversity values for the Simpson index because it would rely on the proliferation of this specific community for its high values. Consequently however, if low values are observed for the Shannon function for the same set of samples, then this would point towards the data being skewed because if one microbe is specifically proliferating in a nutrient-rich environment, then there should be some other microbes that should be proliferating as well. The fact that Shannon diversity is only high for the sample that contained all of the *Bacteroides* in Figure 8, seems to explicate that the loss of sequences from either sampling or processing methods, skewed our results. Furthermore, the trends apparent in oral samples ceases to exist in Figure 8 except for the artifically created low oral sample from body006 from 08Sep2011 (the sample was discarded during processing of results because of insufficient abundances).

Future Steps
------------

It became very apparent during the course of the study, that better methods of decomposition progress characterization are needed in order to be able to keep sampling consistent even when certain organs are decomposed or are no longer recognizable. Furthermore, better methods in actual sampling are also needed, as there is some suspicion that samples were cross-contaminated. If these specifics can be addressed, then it is very likely that A PMI using microbial sequencing data will be able to be developed in order to determine the time since death of a human being. Specifically, the putrefaction and bloat stages undergoe a natural transition that might be able to be correlated with the time of death. This natural pattern exhibits an important theme in analyzing Bioinformatic data, as we can rely on these natural processes to develop algorithms and observations of our own.

Sources Cited
=============

Carter,D.O. *et al.* (2008) Temperature affects microbial decomposition of cadavers (Rattus rattus) in contrasting soils. *Applied Soil Ecology*, **40**, 129–137.

Cockle,D.L. and Bell,L.S. (2015) Human decomposition and the reliability of a ’Universal’ model for post mortem interval estimations. *Forensic Science International*, **253**, 136.e1–136.e9.

Comparison of decomposition rates between autopsied and non-autopsied human remains (2016) *Forensic Science International*, **261**, 93–100.

Distinctive thanatomicrobiome signatures found in the blood and internal organs of humans (2014) *Journal of Microbiological Methods*, **106**, 1–7.

Emmons,A.L. *et al.* (2017) The persistence of human DNA in soil following surface decomposition. *Science and Justice*, **57**, 341–348.

Ferreira,M.T. and Cunha,E. (2013) Can we infer post mortem interval on the basis of decomposition rate? A case from a Portuguese cemetery. *Forensic Science International*, **226**, 1–6.

Hauther,K.A. *et al.* (2015) Estimating Time Since Death from Postmortem Human Gut Microbial Communities. *Journal of Forensic Sciences*, **60**, 1234–1240.

Hyde,E.R. *et al.* (2013) The Living Dead: Bacterial Community Structure of a Cadaver at the Onset and End of the Bloat Stage of Decomposition. *PLoS ONE*, **8**, e77733.

Hyde,E.R. *et al.* (2015) Initial insights into bacterial succession during human decomposition. *International Journal of Legal Medicine*, **129**, 661–671.

Javan,G.T. *et al.* (2017) Cadaver thanatomicrobiome signatures: The ubiquitous nature of Clostridium species in human decomposition. *Frontiers in Microbiology*, **8**, 1–10.

Megyesi,M. *et al.* (2005) Using accumulated degree-days to estimate the postmortem interval from decomposed human remains. *J Forensic Sci*, **50**, 618–626e.

Metcalf,J.L. *et al.* (2016) Mammalian Corpse Decomposition. *Science*, **351**, 158–162.

Protectingusnow.org (2017) Student Handout: How to Calculate Biodiversity. 1–4.

Ríos-Covián,D. *et al.* (2016) Intestinal short chain fatty acids and their link with diet and human health. *Frontiers in Microbiology*, **7**, 1–9.

Rosier,E. *et al.* (2015) The search for a volatile human specific marker in the decomposition process. *PLoS ONE*, **10**, 1–15.

The thanatomicrobiome: A Missing Piece of the Microbial Puzzle of Death (2016) *Frontiers in Microbiology*, **7**.

Vass,A. (2001) Beyond the grave – understanding human decomposition. *Microbiology Today*, **28**, 190–192.

Vass,A.A. (2011) The elusive universal post-mortem interval formula. *Forensic Science International*, **204**, 34–40.
