---
title: "Final Report: Life After Death- Identifying microbiotic signatures in decomposing cadavers"
author: "Alfredo H. Locht"
date: "14Dec2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---
```{r library setup and phyloseq_obj load; will stop if phyloseq_obj does not load include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library("dplyr")
library("tidyr")
library("knitr")
library("phyloseq")
library("ggplot2")
load("output/Thanatomicrobiome_phyloseq.RData")

stopifnot(exists("phyloseq_obj"))
```

# Introduction

The decomposition of the human body can yield many clues which forensic scientists and law enforcement use [@Hauther2015] to solve criminal investigations. In the first four minutes after a human dies, the body is depleted of oxygen and carbon dioxide begins to dissolve in the blood, creating carbonic acid and lowering the pH of the blood, causing cells to lyse (autolysis) and release their contents [@Vass2001; @Vass2011]. These include many enzymes such a proteases and lipases which have digestive activities that begin the initial breakdown of bodily tissue and organs [@Vass2001; @Can2014; @Cockle2015]. Body organs which have high enzymatic activity such as the liver, or provide an aqueous environment that aids in providing the necessary ions for enzymatic catalysis such as the brain, tend to decompose faster than other organs, so there is a wide variability of rates at which specific sites on the body tends to decompose; however, the body can be characterized at each of the different stages of decomposition in summation [@Hyde2013; @Can2014; @Hyde2015]. The five stages of human decomposition include: Fresh which will be termed Autolysis, Putrefaction/Bloat, Purge (Active Decay), Advanced Decay and Skeletonization. The rate at which is one occurs or lasts is highly dependent on a myriad of factors that up to date, have yet not been efficiently addressed [@Hyde2013; @Hyde2015].
 
Characterization of each step in the decomposition process is critical to law enforcement, forensic scientists and many other disciplines alike because they use these markers to reveal information about the Post Mortem Interval (PMI), or better known as the time since death [@Bates2016; @Ferreira2013; @Hauther2015; @Hyde2013]. The time since death is a key contributor to solving investigations surrounding death because it will provide a c The PMI is obtained in a variety of ways using a variety of methods, but has previously relied on visual markers, chemical profile analyses,  measurement of enzymatic activity, carbon dioxide-carbon evolution and other predominantly visual or chemically-based methods of analyses [@Carter2008; @Ferreira2013; @Hauther2015; @Rosier2015; @Vass2011].

Each method of course, has its corresponding error rate which can overestimate or underestimate the PMI and lead to erroneous results [@Cockle2015]. For example, very frequently, validation of a specific PMI calculation derived by one set of researchers in a particular geographic location is only valid at that particular location because when tested in significantly different ecological and temperature climate, the studies cannot match the results expected or previously reported [@Cockle2015].  Furthermore, researchers often argue about which factors directly influence the PMI, so a consensus on how to develop methods which would measure certain sets of factors, has not been reached [@Carter2008; @Cockle2015; @Ferreira2013]. It also does not help that the identified list of potential factors influencing PMI estimations include endogenous factors such as body and fat mass, penetrating trauma, drug use, disease profile, and clothing as well as exogenous factors such as temperature, vegetation in the area, humidity, type of soil, and scavenger and insect access all compile to precisely paint the rate and pattern of a decomposition picture [@Bates2016; @Carter2008; @Cockle2015; @Hauther2015]. What this picture is missing however, are the organisms directly responsible for the process.

Interestingly, microbial communities have been outright dismissed by certain forensic scientists as having anything to do with the decomposition process of human corpses [@Carter2008], yet other sources explain that microbial decomposers are considered to have co-evolved with the presence of vertebrate corpses for the need to recycle complex biological molecules back into basic compounds and elements [@Metcalf2016]. 


# Methods

## Study design

[@Hyde2013] obtained cadavers donated to the Southeast Texas Applied Forensic Science (STAFS) Facility at the Center for Biological Field Studies (CBFS) at Sam Houston State University (SHSU). Both CBFS and STAFS are located in a humid, subtropical climate characterized by a large distribution of pine trees and acidic soils in an area 5 km North of Huntsville, Texas. STAFS is a sectioned-off, two-acre area within the larger, 243 acre CBFS complex which is covered with pine trees and herbaceous plants. The STAFS facility is considered to be a “willed donations” center, so donors or family members of the deceased are required to sign a form in which they indicate that they understand that their donated corpse will be used for scientific research and teaching purposes. 

This form also included information regarding the methods that would be used to study the cadavers. Specifically, the study informed the donors and/or next of kin that the body would be left outside to decompose in natural and aerobic conditions, and that samples pertaining to different types of research (bacterial, tissue, and entomological) would be acquired at different stages of the decomposition to support the associated research. Upon request, the family members could also choose to be debriefed about the scientific knowledge or educational project that was achieved during the course of the study and as a result of the donated cadaver. 

## Sample Origin and Sequencing

### Cadaver Demographics

The two cadavers donated to the center that were studied, STAFS 2011-016 and STAFS 2011-006, were both males of ages 68 and 52 respectively, both of Caucasian descent, were corpses that had not been autopsied, and were considered to be in the early stages of decomposition according to a qualitative body-scoring index developed by [@Megyesi2005]. This index relied on visual observation of the cadavers for signs of the bloat process [@Hyde2013]. STAFS 2011-016 had a previous medical history relating to the cause of death (acute myocardial infarction), been frozen for 89 days, and had been subsequently defrosted in a fridge cooler for 143 days before being placed outside to decompose. STAFS 2011-006 did not have any pertaining medical history other than the cause of death (carbon monoxide poisoning), had been frozen for 13 days, and had subsequently defrosted for 6 days in a fridge cooler before decomposition placement. The time since death was unknown for both cadavers. 

### Sampling Methods

Bacterial sampling of each cadaver was obtained from multiple body sites at two different time points, Placement (Pre-Bloat Phase) and Conclusion (End-Bloat Phase). The Pre-Bloat samples were taken right before the corpse was placed in a supine position outside with no clothing to decompose in natural conditions, while the End-Bloat samples required that the cadaver be dissected in order to gain access to the internal organs. Because this procedure was intrusive and compromised the structural integrity of the cadaver, it marked the end time-point of the sampling. 

Pre-bloat samples were taken as either swabs or “scrapes” from the mouth and rectum by swabbing/scraping with a sterile cotton applicator which was subsequently cut into a sterile PBS solution and prepped for sequencing using the PowerSoil DNA Isolation Kit (MoBio). Bloat samples were taken from the corpses at a time which they were identified to have purged their bodily fluids according to the body scoring system [@Megyesi2005]. Maggots found on the cadavers were also removed at this point with sterile plastic spatulas for sampling purposes. 

The samples obtained from different parts of the body or organs were sampled whenever those body parts or organs still existed. Specifically, STAFs 2011-016 no longer retained any evidence of a sigmoidal colon at the end-bloat sampling.

### Limting Disclaimers

Conditions surrounding the death of any given cadaver can have high variability in confounding factors which can directly influence the rate and pattern of decomposition [@Bates2016; @Cockle2015; @Ferreira2013; @Hauther2015; @Hyde2013; @Vass2011]. Intrinsic factors such as cause of death, physical trauma, and postmortem care can directly interact with a number of other, extrinsic factors such as temperature and humidity of environment in manners which have not yet been efficiently characterized [@Cockle2015]. [@Hyde2013] forewarn that because of the inherent inconsistency of the STAFS willed donations program (cadavers can be obtained on a random or unexpected basis) and the associated complications in conducting decompositional studies on cadavers, data acquired for both cadavers cannot be compared directly but rather, should act as a complimentary set of data. Although the authors sought to control or reduce the amount of confounding variables to the best of their knowledge, they were able to identify some of the limiting qualities of their study. These disclaimers are investigated below: 

In order to establish a baseline of decomposition of both cadavers, they were initially qualified by a visual scoring system developed by [@Megyesi2005]. A summary of the system includes visible cues that might indicate at what step of the decomposition process a particular cadaver may be at. Using this qualitative method, both cadavers were considered to be in the early stages of decomposition as no bloat had yet occurred. This system was also used to identify the end-point of sampling when the purging of bodily fluids occurs; however, due to its qualitative nature, the cadavers did not spend an equal amount of time decomposing outside. 

STAFS 2011-006 spent seven days out in September while STAFS 2011-016 spent fourteen days out in November. Because the body scoring system developed by [@Megyesi2005] is completely dependent on temperature, [@Hyde2013] retroactively calculated the heat-energy units for each cadaver which are a function of Accumulated Degree-Days (ADD), an arbitrary calculation of how much thermal energy a sample absorbs during a given time period, a maximum temperature and a chosen starting point. The results of the calculations were 197.14 heat-energy units for STAFS 2011-006 and 199.97 heat-energy units for STAFS 2011-016 when a thermal minimum of 4 degrees Celsius was chosen. Because November temperatures in Texas are less than September temperatures, it is logical to see that STAFS 2011-016 would probably have to spend twice as long outside in November to achieve the same level of decomposition as STAFS 2011-006 in September, but again, this is based on a visual and qualitative scoring index. Furthermore, the authors mention that choosing a different thermal minimum would result in differing calculations from the ones above [@Hyde2013].

Other variables speaking towards the physical characteristics such as height and weight of each male cadaver were unknown, but estimated to be of average height and above-average weight in comparison to an average American male [@Hyde2013].

# Results

```{r melt phyloseq object for dplyr}
melted_obj <- psmelt(phyloseq_obj)
```

### Figure 1: Alpha Diversity (Shannon and Simpson Index)
```{r alpha diversity measure all, message = FALSE, echo = FALSE, warning = FALSE}
plot_richness(phyloseq_obj,
              x = "sample_description_s",
              measures = c("Shannon", "Simpson"),
              color = "collection_date_s") +
  xlab("Source of Sample") +
  geom_jitter(width = 0.2) +
  theme_dark() +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Shannon and Simpson Alpha Diversity")
```
Figure 1- Alpha diversity of the entire sample set, calculated by the Shannon function and Simpson index is displayed in Figure 1. Overall values for both measures of diversity are high for most samples except for in body006 and body016 cavity samples. Note the trends in the data which show that diversity for oral samples in body006 increase while diversity for oral samples in body016 decrease. These are important baselines to consider and establish because each cadaver has a significantly different starting diversity While no comparison addressing differences or similarites between cadavers will be made in the rest of the study, individualistic trends will be assessed to attempt to understand how the microbial communities of these two cadavers changes over time.

### Table 1: OTUs with Abundance > 1
```{r 100 most abundant samples, message = FALSE, echo = FALSE, warning = FALSE}
topabundant<- melted_obj %>%
  group_by(Sample, Abundance, collection_date_s, sample_description_s, Phylum, Family, Class, Genus) %>%
  summarize(mean_count = mean(Abundance)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 289)
 
topabundant %>%
  kable()
```

### Figure 2: Firmicutes Phyla Abundance
```{r abundance of firmicutes phyla across time,message = FALSE, echo = FALSE, warning = FALSE}
topabundant %>%
  filter(Phylum == "Firmicutes") %>%
ggplot(aes(x = sample_description_s,
           y = Abundance,
           fill = collection_date_s)) +
    geom_col(position = "dodge") +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Firmicutes Phyla Abundance")
```
Figure 2 Description:

### Figure 3: Firmicutes Genera at Pre and End Bloat
```{r changes in firmicutes genera, message = FALSE, echo = FALSE, warning = FALSE}
topabundant %>%
  filter(Phylum == "Firmicutes") %>%
ggplot(aes(x = collection_date_s,
           y = Abundance,
           fill = Genus)) +
    geom_col(position = "stack") +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Firmicutes Genera at Pre and End Bloat")
```
Figure 3 Description:

### Figure 4: Comparison of Firmicutes Classes per Sample Type
```{r firmicutes classes of different sample types, message = FALSE, echo = FALSE, warning = FALSE}
topabundant  %>%
  filter(Phylum == "Firmicutes") %>%
ggplot(aes(x = collection_date_s,
           y = Abundance,
           fill = sample_description_s)) +
    geom_col(position = "dodge") +
  facet_grid(~Class) +
  theme_dark() +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
ggtitle("Comparison of Firmicutes Classes")
```
Figure 4 Description:


### Figure 5: Comparison of Firmicutes Classes at Pre and End Bloat
```{r visual ratios of different classes in the firmicutes phyla, message = FALSE, echo = FALSE, warning = FALSE}
topabundant %>%
  filter(Phylum == "Firmicutes") %>%
ggplot(aes(x = collection_date_s,
           y = Abundance,
           fill = Class)) +
    geom_col(position = "stack") +
  facet_grid(~sample_description_s) +
  theme_dark() +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Comparison of Firmicutes Classes Ratios")
```
Figure 5 Description: 


### Figure 6- Bacilli Class Alpha Diversity
```{r bacilli class alph diversity over time, message = FALSE, echo = FALSE, warning = FALSE}
basil <- subset_taxa(phyloseq_obj, Class == "Bacilli")
plot_richness(basil,
              x = "sample_description_s",
              measures = c("Shannon","Simpson"),
              color = "collection_date_s") +
  xlab("Source of Sample") +
  geom_jitter(width = 0.2) +
  theme_dark() +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Bacilli Alpha Diversity")
  
```
Figure 6 Description:  Shows that diversity of bacilli were mainly low in body006 specific organs, but remained high in the body cavity. For body016, diversity remained high.

### Figure 7: Proteobacteria Phyla Abundance
```{r abundance of proteobacteria phyla,message = FALSE, echo = FALSE, warning = FALSE}
topabundant %>%
  filter(Phylum == "Proteobacteria") %>%
ggplot(aes(x = sample_description_s,
           y = Abundance,
           fill = collection_date_s)) +
    geom_col(position = "dodge") +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Abundance of Proteobacteria Phyla")
```
Figure 7 Description: 

### Figure 8: Proteobacteria Families per Sample Type
```{r comparison of proteobacteria families per sample type, message = FALSE, echo = FALSE, warning = FALSE}
topabundant %>%
  filter(Phylum == "Proteobacteria") %>%
ggplot(aes(x = collection_date_s,
           y = Abundance,
           fill = Family)) +
    geom_col(position = "dodge") +
  facet_wrap(~sample_description_s) +
  theme_dark() +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Comparison of Proteobacteria Families")
```
Figure 8 Description: 

### Figure 9: Bacteroidetes Genera Abundance
```{r abundance of bacteroidetes phyla across time,message = FALSE, echo = FALSE, warning = FALSE}
topabundant %>%
  filter(Phylum == "Bacteroidetes") %>%
ggplot(aes(x = sample_description_s,
           y = Abundance,
           fill = Genus)) +
    geom_col(position = "dodge") +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Bacteroidetes Genera Abundance")
```
Figure 9 Description: 



### Figure 10: Bacteroidetes Phyla Alpha Diversity
```{r bacteroidetes phyla shannon and simpson diversity, message = FALSE, echo = FALSE, warning = FALSE}
steroid <- subset_taxa(phyloseq_obj, Phylum == "Bacteroidetes")
plot_richness(steroid,
              x = "sample_description_s",
              measures = c("Shannon","Simpson"),
              color = "collection_date_s") +
  xlab("Source of Sample") +
  geom_jitter(width = 0.2) +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  theme_dark()
```
Figure 10 Description:












Figure 2- Shows Bacilli Class Distribution across samples. Of interest, Staphylococcaceae are only present in one sample, SRR1016419, a body016 sample taken orally on 03Nov2011 at the pre-bloat sampling.




---
#Simpson's index is a similarity index (the higher the value the lower in diversity). If you want to use it as a diversity index you can subtract it to 1 (i.e. 1-S). Conversely, the value of the Shannon function (it is not an index) increases as diversity increases.

#The Shannon index is an information statistic index, which means it assumes all species are represented in a sample and that they are randomly sampled. Can you point out any problems in these assumptions?

#The Simpson index is a dominance index because it gives more weight to common or dominant species. In this case, a few rare species with only a few representatives will not affect the diversity. Can you point out any problems in these assumptions?

---





# Discussion
These are important baselines to consider and establish because each cadaver has a significantly different starting diversity While no comparison addressing differences or similarites between cadavers will be made in the rest of the study, individualistic trends will be assessed to attempt to understand how the microbial communities of these two cadavers changes over time.

# Sources Cited
