---
title: "Final Report: Life After Death- The Prevailing Microbial Species of Recently-Deceased Individuals"
author: "Alfredo Hans Locht"
date: "November 29, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("png")
load("output/Thanatomicrobiome_phyloseq.RData")

```

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design

[@Hyde2013] obtained cadavers donated to the Southeast Texas Applied Forensic Science (STAFS) Facility at the Center for Biological Field Studies (CBFS) at Sam Houston State University (SHSU). Both CBFS and STAFS are located in a humid, subtropical climate characterized by a large distribution of pine trees and acidic soils in an area 5 km North of Huntsville, Texas. STAFS is a sectioned-off, two-acre area within the larger, 243 acre CBFS complex which is covered with pine trees and herbaceous plants. The STAFS facility is considered to be a “willed donations” center, so donors or family members of the deceased are required to sign a form in which they indicate that they understand that their donated corpse will be used for scientific research and teaching purposes. 

This form also included information regarding the methods that would be used to study the cadavers. Specifically, the study informed the donors and/or next of kin that the body would be left outside to decompose in natural and aerobic conditions. Upon request, the family members could also choose to be debriefed about the scientific knowledge or educational project that was achieved during the course of the study and as a result of the donated cadaver. 

## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Computational

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results

```{r}
plot_richness(phyloseq_obj,
              x = "collection_date_s",
              measures = c("Shannon", "Simpson"),
              color = "sample_description_s") +
  xlab("Source of Sample") +
  geom_jitter(width = 0.2) +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1))
  theme_bw()
```

```{r}
plot_richness(phyloseq_obj,
              x = "sample_description_s",
              measures = c("Shannon", "Simpson"),
              color = "collection_date_s") +
  xlab("Source of Sample") +
  geom_jitter(width = 0.2) +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1))
  theme_bw()
```

```{r}
gp_ch <- subset_taxa(phyloseq_obj, Phylum == "Firmicutes")
plot_bar(gp_ch,
  fill = "Family",
  title = "Firmicutes Phyla Acrosss Distribution Across Samples")
```

```{r}
plot_richness(gp_ch,
              x = "sample_description_s",
              measures = c("Shannon", "Simpson"),
              color = "collection_date_s") +
  xlab("Source of Sample") +
  geom_jitter(width = 0.2) +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1))
  theme_bw()
```





```{r}
gp_ch2 <- subset_taxa(phyloseq_obj, Phylum == "Actinobacteria")
plot_bar(gp_ch2,
  fill = "Family",
  title = "Actinobacteria Phyla Acrosss Distribution Across Samples")
```


```{r}
gp_ch3 <- subset_taxa(phyloseq_obj, Class = "Chlostridia")
plot_bar(gp_ch3,
  fill = "Order",
  title = "Firmicutes Phyla Acrosss Distribution Across Samples")
```

```{r}
gpac <- subset_taxa(phyloseq_obj, Phylum == "Firmicutes")
gpac <- prune_taxa(names(sort(taxa_sums(gpac),TRUE)[1:300]), gpac)
plot_heatmap(phyloseq_obj,
             method = "NMDS"
             )









```











## Subsections are ok in the results section too

```{r your-code-here}
# Add code chunks as needed for your analyses
# For most analyses, I would recommend splitting the
# intensive computational part into a seperate R script file
# and then just load your libraries and the data object here
# using the `load()` function. Ask me for clarification if this is
# unclear.
```

In addition to a minimum of 5-10 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
